<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DmxDumpFactory</class>
 <widget class="QDialog" name="DmxDumpFactory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dump DMX values</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qlcui.qrc">
    <normaloff>:/add_dump.png</normaloff>:/add_dump.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QCheckBox" name="m_nonZeroCheck">
     <property name="text">
      <string>Dump only non-zero values</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Dump options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QRadioButton" name="m_dumpAllRadio">
        <property name="text">
         <string notr="true">Dump all channels</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="m_dumpSelectedRadio">
        <property name="text">
         <string>Dump selected channels</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QTreeWidget" name="m_fixturesTree">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Type</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QTreeWidget" name="m_chasersTree">
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Add to Chaser:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Scene name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_sceneName"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="qlcui.qrc"/>
  <include location="qlcui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DmxDumpFactory</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DmxDumpFactory</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_dumpAllRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>DmxDumpFactory</receiver>
   <slot>slotDumpModeChanged(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>809</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_nonZeroCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>DmxDumpFactory</receiver>
   <slot>slotDumpNonZeroChanged(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>800</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotDumpModeChanged(bool)</slot>
  <slot>slotDumpNonZeroChanged(bool)</slot>
 </slots>
</ui>
